src/Data.ts

```diff
@@ -34,7 +34,7 @@ export class Variable {
   }) {
     Object.assign(this, options);
```

Comment:
In the `Costume` constructor, I had to nullish-coalesce `centerX` and `centerY` since they can be null but not undefined, whereas here, there's no special handling needed for any of the options. I'd like to remove the `Object.assign` pattern entirely in a future PR.

Reply:
Got it, sounds good.